pypeit/pypeit.py

```diff
@@ -245,7 +245,9 @@ def get_std_outfile(self, standard_frames):
         # isolate where the name of the standard-star spec1d file is defined.
         std_outfile = self.par['reduce']['findobj']['std_spec1d']
         if std_outfile is not None:
-            if not Path(std_outfile).absolute().exists():
+            if not self.par['reduce']['findobj']['use_std_trace']:
```

Comment:
can we do this vetting in `pypeitpar.py` ?

Reply:
It is actually also done in `pypeitpar.py`. It may be redundant but I was just following what already existed, i.e., vetting the existence of the file here and in `pypeitpar.py`

Reply:
Unfortunately, `validate` is only called during instantiation.  So if the parameter is changed by the code after the `ParSet` is instantiated (e.g., by the `default_pypeit_par` or `config_specific_par` spectrograph functions), the validation won't catch a file that doesn't exist.  Something to refactor (in infinite time...).